samples/streams/atLeast/README.md

```diff
@@ -0,0 +1,28 @@
+# Hello World Example
+
+To run the example on http://localhost:8080
+
+```sh
+$ ./workerd serve config.capnp
+```
+
+To run using bazel
+
+```sh
+$ bazel run //src/workerd/server:workerd -- serve ~/cloudflare/workerd/samples/streams/atLeast/config.capnp
```

Comment:
I don't feel strongly about this at all, but you might consider kebab-case rather than camelCase for `atLeast`. I think that's our prevailing style for filenames.